cradle-core/src/main/java/com/exactpro/cradle/testevents/BatchedStoredTestEventBuilder.java

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright 2021-2022 Exactpro (Exactpro Systems Limited)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.exactpro.cradle.testevents;
+
+import com.exactpro.cradle.PageId;
+
+import java.time.Instant;
+
+public class BatchedStoredTestEventBuilder {
```

Comment:
Is this class only useful for tests? If so, it would be better to move it to test sources, I think.

Reply:
No it is used in com.exactpro.cradle.serialization.EventBatchDeserializer#deserializeBatchEntry(java.nio.ByteBuffer, com.exactpro.cradle.serialization.EventBatchCommonParams)